EtherKit/Models/Signature.swift

```diff
@@ -23,13 +23,39 @@ public struct Signature: Marshaling {
     ]
   }
 
+  public func toString() -> String {
```

Comment:
Instead of this, just have Signature implement UnformattedDataType protocol (shouldn't require any other things being implemented).

... then you can just use the string constructor: `String(describing: signature)` to get a hex padded String.